src/main/java/com/amihaiemil/docker/RtImages.java

```diff
@@ -87,4 +89,37 @@
             get.releaseConnection();
         }
     }
+
+    // @todo #83:30min Several API calls required an authentication header as
+    //  explained here:
+    //  https://docs.docker.com/engine/api/v1.35/#section/Authentication
+    //  (including Images.create()). Find a way to make a reusable object from
+    //  that action and introduce it here.
+    // @checkstyle ParameterNumber (4 lines)
+    @Override
+    public Images create(
+        final String name, final URL source, final String repo, final String tag
+    ) throws IOException, UnexpectedResponseException {
+        final HttpPost create  = new HttpPost(
+            new UncheckedUriBuilder(this.baseUri.toString().concat("/create"))
+                .addParameter("fromImage", name)
+                .addParameter("fromSrc", source.toString())
+                .addParameter("repo", repo)
+                .addParameter("tag", tag)
+                .build()
+        );
+        try {
+            final int status = this.client.execute(create)
+                .getStatusLine()
+                .getStatusCode();
+            if (HttpStatus.SC_OK != status) {
+                throw new UnexpectedResponseException(
+                    create.getURI().toString(), status, HttpStatus.SC_OK
+                );
+            }
+            return this;
```

Comment:
@llorllale Why return ``this`` here? 

Maybe we could return ``Image`` -- the created image. Since we have its name and, according to the API, we can use the id or name to do operations on an Image. WDYT? If you think it's ok, leave a puzzle for that.

Reply:
@amihaiemil I'm not sure which sequence of commands for the docker API you're referring to.

I see that `search images` is exclusive to Docker Hub.

The docs for `create image` do not show an example response with the image's ID.

The example responses for `list images` do not include the name.

Reply:
@llorllale ok, leave it like this for now, I'll think more about it